python/observability-online-eval/agno/agent.py

```
@@ -0,0 +1,75 @@
+from agno.models.openai import OpenAIChat
+from agno.tools.googlesearch import GoogleSearchTools
+from agno.tools.yfinance import YFinanceTools
+from maxim import Maxim
+from maxim.logger.agno import instrument_agno
+
+# Load environment variables from .env file
+load_dotenv()
+
+instrument_agno(Maxim(
+    {
+        "debug": True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)